.storybook/webpack.config.js

```diff
@@ -1,25 +1,31 @@
-const path = require("path");
+const path = require('path');
 module.exports = ({ config, mode }) => {
-  config.module.rules.push({
-    test: /\.(ts|tsx)$/,
-    loader: require.resolve("babel-loader"),
-    options: {
-      presets: [
-        "@babel/preset-typescript",
-        "@emotion/babel-preset-css-prop",
-        [
-          "@babel/preset-env",
-          {
-            targets: {
-              esmodules: true,
-            },
-          },
-        ],
-      ],
-      plugins: [],
-    },
-  });
-  config.resolve.extensions.push(".ts", ".tsx");
+    config.module.rules.push({
+        test: /\.m?(j|t)sx?$/,
+        loader: require.resolve('babel-loader'),
+        exclude: {
+            test: /node_modules/,
 
-  return config;
+            // don't exclude '@guardian' node_modules
+            exclude: /@guardian\//,
+        },
```

Comment:
Whitespace formatting has broken this diff, apologies. The substantive changes are:

- updating `test` to `/\.m?(j|t)sx?$/` to cover plain TS as well as TypeScript
- Adding `exclude` block to transpile `@guardian` modules

Both changes are in accordance with our [recommendations on @guardian npm packages](https://github.com/guardian/recommendations/blob/main/npm-packages.md#using-guardian-npm-packages)